index.html

```diff
@@ -0,0 +1,162 @@
+                <span class="close">&times</span>
+                <ul>
+                    <li><a href="" class="nav-link">About</a></li>
+                    <li><a href="" class="nav-link">Program</a></li>
+                    <li><a href="" class="nav-link">Join</a></li>
+                    <li><a href="" class="nav-link">Sponsor</a></li>
+                    <li><a href="" class="nav-link">News</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="desktop-version">
+            <ul id="first-nav">
+                <li><a href="#facebook"><img src="images/images/Rectangle.png" alt=""></a></li>
+                <li><a href="#twitter"><img src="images/images/twitter (1).png" alt=""></a></li>
+                <li class="eng">English</li>
+                <li class="eng">My page</li>
+                <li class="eng">Logout</li>
+            </ul>
+            <ul id="second-nav">
+                <li><div class="desk-logo"><img src="images/images/logo.png" alt=""></div></li>
+                <li><a href="" class="nav-link">About</a></li>
+                <li><a href="" class="nav-link">Program</a></li>
+                <li><a href="" class="nav-link">Join</a></li>
+                <li><a href="" class="nav-link">Sponsor</a></li>
+                <li><a href="" class="nav-link">News</a></li>
+                <li class="navbar-link campaign">YLS Campaign</li>
+            </ul>
+        </div>
+    </nav>
+    <section id="homepage">
+        <h2 class="hello hello33">"Hello! Sharing world"</h2>
+        <h1 class="heading">Youth Leadership <br> Summit <br>2023 </h1>
+        <p class="paragraph-1">The annual Youth Leadership Conference is a dynamic event designed to inspire, educate, and empower young individuals. The conference features a range of engaging content and interactive sessions tailored to the needs and interests of youth.</p>
+        <div class="timedate">2023.10.15(THU) ~ 16(FRI)</div>
+        <div class="location">@Serena Hotel of Kenya, Hilton Conference Hall and more</div>
+    </section> 
+    <section id="program">
+        <h2 class="list">Main Program</h2>
+        <div class="baseline">.</div>
+        <ul class="mainprogram">
+            <li class="box box1">
+                <ul>
+                    <li class="activity ignite">Lecture</li>
+                    <li class="angagement">Listen to the speakers from various countries about the messages of sharing and opening.</li>
+                </ul>
+            </li>
+            <li class="box box2">
+                <ul>
+                    <li class="activity ignite1 ignite">CC <br> exhibition</li>
+                    <li class="activity exhibition">CC exhibition</li>
+                    <li class="angagement angagement1">Witness the impactful projects and initiatives undertaken by young leaders and organizations.</li>
+                </ul>
+            </li>
+            <li class="box box3">
```

Comment:
> All page elements have similar spacing, borders, shadows, and paddings than the orignal design.

- [ ] Looking at the given design, you can see that when a program card has hover, there is a white border around the card, this is not currently implemented in your program cards, so I'll suggest you do so. :+1: :+1:  I'll also suggest you canter the icons in your own implementation as they are currently aligned to the left. :+1: 

- - Your design:
![Selection_001](https://github.com/denigit5/First-Capstone/assets/107632780/5c40e8b3-6841-45a2-aa07-829648b5c6a0)


- - Provided design:
 
![Screenshot from 2023-08-08 23-38-33](https://github.com/IvonneBenitesRodriguez/capstone-project/assets/107632780/c31ea879-76e9-4319-a586-7486869be1de)

- [ ] Also looking at the screenshot below, you can see that the line under the text `SEE THE WHOLE PROGRAM` tends to be longer than the content, which is not how it appears on the design, I'' suggest you also adjust this correctly. :smile: :wave: 

- - screenshot:
![Screenshot from 2023-08-10 10-10-11](https://github.com/denigit5/First-Capstone/assets/107632780/03ef8531-75eb-4007-ab7a-504ccc74a8ac)

Reply:
As requested by the previous code reviewer it would be nice if you implemented this change. As explained by the previous code reviewer:


 All page elements have similar spacing, borders, shadows, and paddings than the orignal design.

  

- [ ]   Looking at the given design, you can see that when a program card has hover, there is a white border around the card, this is not currently implemented in your program cards, so I'll suggest you do so. 👍 👍 I'll also suggest you canter the icons in your own implementation as they are currently aligned to the left. 👍

This is your design:
![259697491-5c40e8b3-6841-45a2-aa07-829648b5c6a0](https://github.com/denigit5/First-Capstone/assets/79506105/1e011d67-2f56-48ad-a7ff-283bffda350c)

Please stick to this design:

![259252984-c31ea879-76e9-4319-a586-7486869be1de](https://github.com/denigit5/First-Capstone/assets/79506105/74088768-2276-4734-a526-9815dc7ed4e7)


 

- [ ] Also looking at the screenshot below, you can see that the line under the text SEE THE WHOLE PROGRAM tends to be longer than the content, which is not how it appears on the design, I'' suggest you also adjust this correctly. 😄 👋

![259692485-03ef8531-75eb-4007-ab7a-504ccc74a8ac](https://github.com/denigit5/First-Capstone/assets/79506105/86aa11ec-b4d5-4c85-a01d-0ef6fadc826d)